msmbuilder/commands/featurizer.py

```diff
@@ -169,6 +170,26 @@ def _contacts_type(self, val):
         else:
             return np.loadtxt(val, dtype=int, ndmin=2)
 
+class BinaryContactFeaturizerCommand(FeaturizerCommand):
+    _concrete = True
+    klass = BinaryContactFeaturizer
+
+    def _contacts_type(self, val):
+        if val is 'all':
```

Comment:
http://stackoverflow.com/questions/1504717/why-does-comparing-strings-in-python-using-either-or-is-sometimes-produce